### DISCUSSION TITLE
Formatting for image files

### DISCUSSION THREAD

**Discussion by pmackay** [avo-hq/avo#1775] [Help]
Am trying to add some Active Storage fields to a type and render in Avo. With an uploaded image, I see:

- The filename displayed on list view
- The image stretched to fill the space (its only a small logo) on the show view

It is possible to:

- Display a thumbnail on list view?
- Prevent stretching on the show view?

Thanks!

**Comment by adrianthedev** [comment]
Hey @pmackay. Thanks for writing.

> Display a thumbnail on list view?

We don't have that feature yet, but we'd welcome a PR. I think [this](https://github.com/avo-hq/avo/blob/main/app/components/avo/fields/common/files/view_type/list_component.html.erb#L1) is the file that should be updated in order to have that.

> Prevent stretching on the show view?

Yes. You can prevent that by changing element display using CSS.
You should first add your own CSS to Avo using [this](https://docs.avohq.io/2.0/custom-asset-pipeline.html#use-tailwindcss-utility-classes) guide and command. Then, target that element in CSS and change it. 
You should have all the required data attributes to target that element. Check out [this guide](https://docs.avohq.io/2.0/recipes/hide-field-labels.html) for reference.
